javascript/rest-nextjs/package.json

```diff
@@ -13,11 +13,11 @@
   },
   "dependencies": {
     "@prisma/client": "4.13.0",
-    "next": "^13.0.0",
+    "next": "13.0.0",
     "next-superjson": "0.0.4",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-markdown": "^8.0.0",
+    "react": "18.0.0",
+    "react-dom": "18.0.0",
+    "react-markdown": "8.0.0",
     "superjson": "1.12.3"
```

Comment:
The pinning here is radical, because it brings us back many versions potentially but it's ok as long as we go up again once Renovate picks up the updates I think

Reply:
Yep, it looks bad but my idea here was to let renovate handle the updating of pinned versions. After the whole node 14/16 with Next conversation.